File: fudi-store/src/test/java/com/romellfudi/sharepreference/resources/ObjectDetailBean.java
package com.romellfudi.sharepreference.resources;

import org.codehaus.jackson.annotate.JsonIgnoreProperties;

import java.util.ArrayList;

/**
 * Created by romelldominguez on 5/15/16.
 */
@JsonIgnoreProperties(ignoreUnknown = true)
public class ObjectDetailBean {

    private String idNotHasSetter;

    private String name;

    private ArrayList<String> details;

    public ObjectDetailBean() {

    }

    public ObjectDetailBean(String id) {
        this.idNotHasSetter = id;
    }

    public ObjectDetailBean(ArrayList<String> details) {
        this.details = details;
    }

    public ArrayList<String> getDetails() {
        return details;
    }

    public void setDetails(ArrayList<String> details) {
        this.details = details;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getId() {
        return idNotHasSetter;
    }
}

File: fudi-store-kotlin/src/test/java/com/romellfudi/sharepreference/resources/ObjectDetailBean.kt
package com.romellfudi.sharepreference.resources

import org.codehaus.jackson.annotate.JsonIgnoreProperties
import java.util.*

/**
 * Created by romelldominguez on 5/15/16.
 */
@JsonIgnoreProperties(ignoreUnknown = true)
class ObjectDetailBean {

    var idNotHasSetter: String = ""
    var name: String? = null

    var details: ArrayList<String>? = null

    constructor() {

    }

    constructor(id: String) {
        this.idNotHasSetter = id
    }

    constructor(details: ArrayList<String>) {
        this.details = details
    }
}
